cmd/ocitools/validate.go

```diff
@@ -84,7 +84,7 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
 		}
 
-		hostCheck := context.Bool("host-specific")
+		hostCheck := hostSpecific || context.Bool("host-specific")
```

Comment:
On Thu, Jul 21, 2016 at 12:31:53PM -0700, hmeng-19 wrote:
> I keep it for now just in case some important users are using it. 

We haven't tagged an ocitools release yet or said anything about
forward compat in it's command-line API, so I think we're free to
change whatever we want.  But if we know of any consumers, giving them
a heads up on the change would be good (ideally as part of a
release-tagging process ;).